mysql-test/t/percona_userstat_threadpool.cnf

```diff
@@ -0,0 +1,15 @@
+!include include/default_my.cnf
```

Comment:
Replace with a -master.opt file that simply has --connection-handling=pool-of-threads